experimental/LieAlgebras/src/LieAlgebras.jl

```diff
@@ -85,120 +85,18 @@ import ..Oscar:
 
 import Base: getindex, deepcopy_internal, hash, issubset, iszero, parent, zero
 
-export AbstractLieAlgebra, AbstractLieAlgebraElem
-export DirectSumLieAlgebra, DirectSumLieAlgebraElem
-export DualRootSpaceElem
-export LieAlgebra, LieAlgebraElem
-export LieAlgebraHom
-export LieAlgebraIdeal
-export LieAlgebraModule, LieAlgebraModuleElem
-export LieAlgebraModuleHom
-export LieSubalgebra
-export LinearLieAlgebra, LinearLieAlgebraElem
-export RootSpaceElem
-export RootSystem
-export WeightLatticeElem
-export WeylGroup, WeylGroupElem
-export WeylOrbitIterator
+include("exports.jl")
 
+# only export into Oscar, not from Oscar
```

Comment:
I find this hard to understand. Perhaps you meant this?
```suggestion
# The following export statements export things *into* module Oscar;
# they are however for internal use and thus are not exported *from* Oscar.
```

Could we perhaps instead replace these by `import` statements, i.e. after module `LieAlgebras` is closed, do:
```
import LieAlgebras: _is_direct_sum
```
and so on? That way nothing would have to be explained. But maybe this doesn't work for some technical reason I am not aware of? (Also: of course this is not a required change for this PR, just a thought)

Reply:
There is a semantical difference (for users, not for the compiler) between exports and imports. This was e.g. the reason that I revamped the in/export between AA and AA.Generic in https://github.com/Nemocas/AbstractAlgebra.jl/pull/1537. And there is tooling like https://github.com/ericphanson/ExplicitImports.jl that suggest to change imports, but they won't do anything about exports.
Thus, I would like to keep them as exports (at least for now), but I am happy to take your comment suggestion here :)